03/app.js

```diff
@@ -1,58 +1,65 @@
 const books = {
-    '978-83-7278-000-3': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Philosopher\'s Stone',
-            pl: 'Harry Potter i Kamień Filozoficzny',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
-    },
-    '83-7278-007-2': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Chamber of Secrets',
-            pl: 'Harry Potter i Komnata Tajemnic ',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
-    },
-}
+	"978-83-7278-000-3": {
+		author: "J.K. Rowling",
+		title: {
+			en: "Harry Potter and the Philosopher's Stone",
+			pl: "'Harry Potter i Kamień Filozoficzny'",
+		},
+		translator: {
+			en: null,
+			pl: "Andrzej Polkowski",
+		},
+	},
+	"83-7278-007-2": {
+		author: "J.K. Rowling",
+		title: {
+			en: "Harry Potter and the Chamber of Secrets",
+			pl: "Harry Potter i Komnata Tajemnic ",
+		},
+		translator: {
+			en: null,
+			pl: "Andrzej Polkowski",
+		},
+	},
+};
 
-books.getAuthor = function(isbn) {
-    // w tym przypadku this === books
-    
-    if(typeof this[isbn] === 'undefined') {
-        // taki ISBN nie istnieje w moim spisie
-        return null;
-    }
+books.getAuthor = function (isbn) {
+	// w tym przypadku this === books
 
-    const author = this[isbn]['author'];
-    if(author) {
-        // zwracam informacje o autorze
-        return author;
-    }
+	if (typeof this[isbn] === "undefined") {
+		// taki ISBN nie istnieje w moim spisie
+		return null;
+	}
 
-    // brak informacji o autorze
-    return false;
-}
+	const author = this[isbn]["author"];
+	if (author) {
+		// zwracam informacje o autorze
+		return author;
+	}
 
-books.getTitle = function(isbn, lang) {
+	// brak informacji o autorze
+	return false;
+};
 
-}
+books.getTitle = function (isbn, lang) {
+	const title = this[isbn]["title"][lang];
```

Comment:
Warto też sprawdzić elementy pod drodze tj. `this[isbn]` oraz `this[isbn]["title"]` - jeśli ich nie będzie to w kodzie pojawi się błąd odpytując od razu `https://github.com/prettier/prettier-vscode/issues/344`